### ISSUE TITLE
Peer ID should be consistant in all node communication 

### ISSUE THREAD

**Issue by prince-chrismc** [pyrsia/pyrsia#172]
Might be out of scope but I'd of expected this to be a member I guess... It's error prone to "extract" the id in multiple places.

https://github.com/pyrsia/pyrsia/blob/44022d8d429c8b0d31a02c42dd37e612e9ffcba2/pyrsia_node/src/main.rs#L229

_Originally posted by @prince-chrismc in https://github.com/pyrsia/pyrsia/pull/169#discussion_r776107520_

**Comment by prince-chrismc** [comment]
After #169 is merged we will be using the peer id in multiple channels and we "create" the same id in multiple places. 
We should only make the ID once and reuse where needs be.

**Comment by fishseabowl** [comment]
maybe save identity:Keypair to local storage, then load to get PeerID

**Comment by prince-chrismc** [comment]
I think that a great future step. Let's just clean up our code first 👍
